core/controllers/library.py

```diff
@@ -300,7 +300,8 @@ def get(self):
             summaries = (
                 summary_services.get_displayable_exp_summary_dicts_matching_ids(
                     exp_ids,
-                    editor_user_id=editor_user_id))
+                    editor_user_id=editor_user_id,
+                    editor_actions=self.actions))
```

Comment:
No need, I think. If you're bundling up user_id + actions into one thing then I retract my earlier comment.